lib/connect.c

```diff
@@ -142,6 +142,43 @@ timediff_t Curl_timeleft(struct Curl_easy *data,
   return (ctimeleft_ms < timeleft_ms)? ctimeleft_ms : timeleft_ms;
 }
 
+void Curl_shutdown_start(struct Curl_easy *data, int sockindex,
+                         struct curltime *nowp)
```

Comment:
This function is used in two places, both of them set `nowp` to NULL. Do you really need this argument?

Reply:
See comment above.